### ISSUE TITLE
Loss VS iteration, long training time while lose is not stable

### ISSUE THREAD

**Issue by hana9090** [dbolya/yolact#406]
I have limited time to use the GPU I have so what I did:

- Each time train the model for 20K iterations which takes 7 hours on the GPU
- Then resume the training using the checkpoint I got from previous training
- I have 24 RAM on GPU so I use batch size = 5, and I froze batch norm in config:

> 'freeze_bn': True

So what I notice that the loss value is coming up and down even if the iteration is increased. So if you see the mask loss at epoch 72 is greater than loss at epoch 85!

So is this related to the batch size? or freezing batch norm? or resuming the training and each time start from the last checkpoint? is this related to lr?

**at epoch 72**
![image](https://user-images.githubusercontent.com/2920520/77826093-768b0000-711e-11ea-9520-ea72662f5721.png)

**epoch 85** 
![image](https://user-images.githubusercontent.com/2920520/77826059-37f54580-711e-11ea-95ab-dfebec37297d.png)

Thanks in advance
